vipps-recurring-api-faq.md

```diff
@@ -171,7 +174,7 @@ users, as described in
 
 Yes.
 
-Vipps now offers the possibility for merchants to as part of the payment flow in the
+Vipps offers the possibility for merchants to as part of the payment flow in the
```

Comment:
part of the payment flow to request the User's information. For example if you wish to retrieve the user's email as part of a User profile setup.

Reply:
Updated